core/transaction/payload/IssueAsset.go

```diff
@@ -14,9 +18,18 @@ func (a *IssueAsset) Data(version byte) []byte {
 }
 
 func (a *IssueAsset) Serialize(w io.Writer, version byte) error {
+	err := serialization.WriteUint64(w, a.Nonce)
```

Comment:
Cannot judge to err, return err directly.

return serialization.WriteUint64(w, a.Nonce)